docs/PRODUCTION-AZURE-CLUSTER.md

```diff
@@ -0,0 +1,53 @@
+# Creating a Production CKAN Cloud cluster using Azure Cloud Platform
```

Comment:
Please follow the style in the AWS file and connect with the README (both recently modified... :) )

Reply:
AWS file contains info about terraform which is not ready for Azure yet (in this branch).

I modified the doc adding prerequisites and removing not used data.